.github/workflows/validation.yml

```diff
@@ -0,0 +1,38 @@
+name: Validation
```

Comment:
Also, it would be great to remove Travis config in the same PR.

Reply:
done!